spec/controllers/comment_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe App::CommentsController, type: :controller do
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/FilePath: Spec path should end with app/comments_controller*_spec.rb.